src/modules/graphics/graphics.c

```diff
@@ -3930,7 +3953,7 @@ void lovrPassCopyImageToTexture(Pass* pass, Image* image, Texture* texture, uint
   layerOffset += measureTexture(texture->info.format, srcOffset[0], 1, 1);
   size_t pitch = measureTexture(texture->info.format, lovrImageGetWidth(image, srcOffset[3]), 1, 1);
   gpu_buffer* buffer = tempAlloc(gpu_sizeof_buffer());
-  char* dst = gpu_map(buffer, totalSize, 64, GPU_MAP_WRITE);
+  char* dst = gpu_map(buffer, (gpu_size)totalSize, 64, GPU_MAP_WRITE);
```

Comment:
If `totalSize` exceeds the maximum value representable by `gpu_size`, it seems like this would lead to memory safety issues.  Ideally `measureTexture` returns a `gpu_size` and ensures the result fits in one.